controllers/helmchartproxy/helmchartproxy_controller.go

```diff
@@ -279,6 +279,7 @@ func (r *HelmChartProxyReconciler) listInstalledReleases(ctx context.Context, na
 func (r *HelmChartProxyReconciler) aggregateHelmReleaseProxyReadyCondition(ctx context.Context, helmChartProxy *addonsv1alpha1.HelmChartProxy) error {
 	log := ctrl.LoggerFrom(ctx)
 
+	fmt.Printf("Aggregating HelmReleaseProxyReadyCondition\n")
```

Comment:
why the `fmt.Printf` lines here? logs should be enough

Reply:
Forgot to remove debug logs! Side question -- is there a way to get V(2) logs to show up when running unit tests/make test? IIRC I added the print statements to ensure that the messages show up.

Reply:
I don't think so... at least not that I know of